proxy/logging/LogFile.cc

```diff
@@ -185,6 +187,32 @@ LogFile::open_file()
       Debug("log-file", "no readers for pipe %s", m_name);
       return LOG_FILE_NO_PIPE_READERS;
     }
+
+#ifdef __FreeBSD__
+    // we don't do this for FreeBSD
+#else
```

Comment:
Please add a #ifdef for Linux :(

Reply:
or check if F_GETPIPE_SZ is available (defined).

Reply:
it's better.

Reply:
ah really sorry about this one i didn't realize it had been merged. confused why the PR build passed but the other didn't?